custom_platforms.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
         "path": "ios",
         "configuration": {
             "removeRulePatterns": [
-                "\\$\\$|\\$\\@\\$",
+                "^(?!scriptlet.)*\\$\\$|\\$\\@\\$",
```

Comment:
I guess we can use it.

> Also in Adam's message

I discussed it with Adam about the brackets. https://regex101.com/r/78iz17/1 seems works fine.

Reply:
Yes, I think that using `#%#` instead of `scriptlet` will be okay.

By the way, it looks like that brackets are required and it cannot just be `(?!scriptlet.)` or `(?!#%#)`.
But should be something like:
```
^((?!#%#).)*\\$\\$|\\$\\@\\$
```
Otherwise rule like this - `example.com$$div` will be not matched, but it should be, if I'm not wrong.

Reply:
Yep, you were right. I updated the regex here https://regex101.com/r/j8ctg1/1 and the regex below (without double escape) works as expected.
```
^((?!#%#).)*\\$\\$|\\$\\@\\$
```